src/Apps/Article/Components/ArticleBody.tsx

```diff
@@ -25,8 +25,7 @@ import { ArticleAd } from "./ArticleAd/ArticleAd"
 import { ArticleSectionFragmentContainer } from "./ArticleSection"
 import { ArticleSectionAdFragmentContainer } from "./ArticleSectionAd"
 import { OPTIMAL_READING_WIDTH } from "./Sections/ArticleSectionText"
-import { AnalyticsContext } from "System/Analytics/AnalyticsContext"
-import { OwnerType } from "@artsy/cohesion"
+import { Analytics } from "System/Analytics/AnalyticsContext"
```

Comment:
One last minor thing and then 👍 to merge: 
```
import { Analytics } from "System/Analytics/Analytics"
```
Keep it 1:1, more findable to intellisense and less confusing with the context-context

Reply:
What?

Reply:
I'm leavin this one alone; there's other exports here as well like `useAnalyticsContext`

Reply:
Heh, leaving it named `AnalyticsInferredContext`